university-for-the-creative-arts-figures.csl

```diff
@@ -83,7 +83,7 @@
               <choose>
                 <if match="any" variable="URL">
                   <text variable="URL" prefix="At: "/>
-                  <text term="accessed" text-case="capitalize-first" prefix="(" suffix=" on"/>
+                  <text term="accessed" plural="true" text-case="capitalize-first" prefix="(" suffix=" "/>
```

Comment:
@ibadger, I take it you added the `plural="true"` by accident? (there is no plural form of the "accessed" term)